### ISSUE TITLE
Do not require space by the incrementation operator 

### ISSUE THREAD

**Issue by phoe** [y2q-actionman/with-c-syntax#2]
Your reader macro, after collecting all the tokens, can iterate over all of them and check if any of them are of form `FOO++` (same thing for `++FOO` and  `--`). If so, it can split that token into `FOO` and `++`.

This should be enough to type the `++` operator in a more natural style - without any spaces between the variable name and the operator.

**Comment by y2q-actionman** [comment]
I see. I have thought only the reader splits and makes tokens.
 Your suggestion is meaningful. I'll try to consider it. Thank you.

**Comment by phoe** [comment]
> I see. I have thought only the reader splits and makes tokens.

That is correct, the reader will see `FOO++` as a single token. Nobody prevents you from adding a post-processing step though, that will accept raw input from the reader and split `FOO++` into a pair of tokens that are `FOO` and `++` in the result.

**Comment by y2q-actionman** [comment]
Sorry, I am very late for responding.

I implemented a new pass for splitting token including `++`, `--`, or unary `*`, `&`. (commit 7d87cea).
And I rewrote examples in README. The example of "Duff’s Device" now looks prettier.

Additionally, I plan to add some features in future:
- Extending this treatment for some operators; When I write C, I use `!`, `~`, `->` without spaces. I think they are good candidates. 
- Trying to look local bindings using `variable-information` of CLtL2, because I want to use `with-c-syntax` in anywhere.

Thank for your suggestion!
